features/step_definitions/project_urgency_steps.rb

```diff
@@ -0,0 +1,50 @@
+Given(/^I select the project is not urgent$/) do
+  urgency_slug = "not_urgent"
+  @app.proposal_overview_page.add_project_urgency.click
+  @app.urgency_page.submit(
+    option: urgency_slug.to_sym
+  )
+  expect(@app.proposal_overview_page).to have_text("Your project is not urgent")
+end
+
+Given(/^I enter the project urgency as "([^"]*)" with a message of "([^"]*)"$/) do |urgency, message|
+  @app.proposal_overview_page.add_project_urgency.click
+  @app.urgency_page.submit(
+    option: urgency.to_sym
+  )
+  @app.urgency_details_page.submit(
+    details: message
+  )
+  expect(@app.proposal_overview_page).to have_text(message)
```

Comment:
Don't think we should have assertions in the Givens here and in the previous Given

Reply:
Removed and Then created